docs/internals/contributing/writing-code/unit-tests.txt

```diff
@@ -232,6 +232,10 @@ like so::
 
    $ pip install -r tests/requirements/py3.txt  # Python 2: py2.txt
 
+If you are experiencing any problems during the installation of these 
+dependencies, please see our :ref:`troubleshooting <troubleshooting-unit-testing>` 
+page.
```

Comment:
Now it makes more sense! I agree.

I just wanted to separate these instructions from other ones, instead of throwing them into one bag called 'Miscellaneous'. I will rename that section name with the one you have proposed. Indeed, `External Dependencies` sounds like a better option